app/src/services/api/api.ts

```diff
@@ -55,6 +59,18 @@ const useListSourceAttributes = (source: Source) => {
   );
 };
 
+const useListResourceFilters = (mapping?: Resource) => {
```

Comment:
I think I will make a change in the api to allow filtering filters by resource. You won't need that. Do I do it here or in another PR ?